src/components/InventoryGroupDetail/GroupDetailInfo.cy.js

```diff
@@ -0,0 +1,47 @@
+import { mount } from '@cypress/react';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import { getStore } from '../../store';
+import GroupDetailInfo from './GroupDetailInfo';
+
+const mountPage = () =>
+    mount(
+        <Provider store={getStore()}>
+            <MemoryRouter>
```

Comment:
I believe you don't any of those wrappers here

Reply:
Yep, I removed them